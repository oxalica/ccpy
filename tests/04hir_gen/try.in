try:
  raise 1
except Exception as e:
  raise e
