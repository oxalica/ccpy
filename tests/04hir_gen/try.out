Start(Local size = 5):
0: PushExcept %0 <6>
1: %1 = 1
2: Raise %1
3: PopExcept
4: %1 = False
5: JmpIfFalse %1 <17>
6: %2 = __intrinsic__get_global0()
7: %3 = "e"
8: %0 = __intrinsic__dict_set3(%2, %3, %0)
9: %0 = "e"
10: %2 = __intrinsic__get_global0()
11: %3 = "__builtin__global_get"
12: %4 = None
13: %2 = __intrinsic__dict_get3(%2, %3, %4)
14: %0 = __intrinsic__tuple_make_(%0)
15: %0 = __intrinsic__v_call2(%2, %0)
16: Raise %0


